tests/sundials/ida_01_n_timesteps.cc

```diff
@@ -0,0 +1,130 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2017 - 2023 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+#include <deal.II/base/parameter_handler.h>
+
+#include <deal.II/lac/full_matrix.h>
+#include <deal.II/lac/vector.h>
+
+#include <deal.II/sundials/ida.h>
+
+#include "../tests.h"
+
+
+/**
+ * Like the base test, but output the number of time steps performed.
```

Comment:
Is the only change to the last test that you add a print statement that shows the number of steps performed (that's what it looked like)? If that is the case, would you mind just extending the existing test? No reason to duplicate computational time and files.

Reply:
I also reduce the amount of output. It's a separate piece of functionality to test the number of time steps, so I thought that in the spirit of unit tests it gets its own test. I don't feel strongly about it, though. Your call.

Reply:
Like I said. I think it is wasteful (cpu presources, future maintenance cost) to add a test instead of adding a single print statement to an existing test.